src/app/app.component.html

```diff
@@ -7,13 +7,23 @@
     <mat-checkbox [(ngModel)]="CLOSE_MENU_BEHAVIOR">Close settings menu on selection (if disabled it stays open, so the
       user can select/deselect multiple items)</mat-checkbox>
   </p>
+  <p>
```

Comment:
You should remove line 10 - it's too much

Reply:
Sure will remove it.  Thanks!